src/interfaces/filing-interfaces/filingData-interface.ts

```diff
@@ -3,4 +3,5 @@ export interface FilingDataIF {
   filingTypeCode: string;
   entityType: string;
   waiveFees?: boolean;
+  futureEffective?: boolean
```

Comment:
I like the code style where we name booleans "isSomething". Thoughts?

Reply:
totally, gap on my part! Will update

Reply:
shoot, sorry. This property is to tie into the Fee Summary `futureEffective` option property. I knew it had to be named this for a reason :P 
So the update would need to be on that common component for us to change this, i think.

Reply:
Oh right. Never mind then.